test/jit/fixtures_srcs/generate_models.py

```diff
@@ -0,0 +1,178 @@
+import torch
+from fixtures_src import TestVersionedDivTensorExampleV4
+from pathlib import Path
+from typing import Set
+import io
+import zipfile
+import logging
+import sys
+from torch.jit.mobile import _load_for_lite_interpreter, _export_operator_list
+
+logging.basicConfig(stream=sys.stdout, level=logging.INFO)
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+
+"""
+This file is used to generate model for test operator change. Please refer to
+https://github.com/pytorch/rfcs/blob/master/RFC-0017-PyTorch-Operator-Versioning.md for more details.
+
+A systematic workflow to change operator is needed to ensure
+Backwards Compatibility (BC) / Forwards Compatibility (FC) for operator changes. For BC-breaking operator change,
+an upgrader is needed. Here is the flow to properly land a BC-breaking operator change.
+
+1. Write an upgrader in caffe2/torch/jit/operator_upgraders.py file. The softly enforced naming format
+is <operator_name>_<operator_overload>_<start>_<end>. For example, the below example means that div.Tensor at
+version from 0 to 3 needs to be replaced by this upgrader.
+
+```
+div_Tensor_0_3 is added for a change of operator div in pr xxxxxxx.
+Create date: 12/02/2021
+Expire date: 06/02/2022
+@torch.jit.script
+def div_Tensor_0_3(self: torch.Tensor, other: torch.Tensor) -> torch.Tensor:
+    if (self.is_floating_point() or other.is_floating_point()):
+        return self.true_divide(other)
+    return self.divide(other, rounding_mode='trunc')
+```
+
+2. In caffe2/torch/csrc/jit/operator_upgraders/version_map.h, add changes like below.
+You will need to make sure that the entry is SORTED according to the version bump number.
+```
+    {"div.Tensor",
+      {{4,
+        "div_Tensor_0_3",
+        "aten::div.Tensor(Tensor self, Tensor other) -> Tensor"}}},
+```
+
+3. After rebuild PyTorch, run the following command and it will auto generate a change to
+fbcode/caffe2/torch/csrc/jit/mobile/upgrader_mobile.cpp
+
+```
+python pytorch/tools/codegen/operator_versions/gen_mobile_upgraders.py
+```
+
+4. Generate the test to cover upgrader.
+4.1 Switch the commit before the operator change, and add a module in
```

Comment:
Could the test be added in the operator change commit?

Reply:
Yeah the test need to be added with the operator change commit. This script needs to run before the commit to do the operator change though. Let me update this part so it's clearer.